src/script/conversation/ConversationRepository.ts

```diff
@@ -285,15 +283,6 @@ export class ConversationRepository {
       this.conversationState,
     );
 
-    if (supportsMLS()) {
-      // we register a handler that will handle MLS conversations on its own
-      registerMLSConversationVerificationStateHandler(
-        this.onConversationVerificationStateChange,
-        this.conversationState,
-        this.core,
-      );
-    }
-
```

Comment:
This has been moved to the app init method. There was a temporal coupling here, the function was called before the `core` was initialized. 

In the `initApp` function we know that the core has just freshly been init